Dizzy/Screens/Login/AppInfoView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  AppInfoView.swift
+//  Dizzy
+//
+//  Created by stas berkman on 05/06/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+protocol AppInfoViewDelegate: class {
+    func aboutButtonPressed()
+    func termsOfUsButtonPressed()
+    func privacyPolicyButtonPressed()
+    func contactUsButtonPressed()
+}
+
+class AppInfoView: UIView {
+
+    weak var delegate: AppInfoViewDelegate?
+
+    let stackView: UIStackView = UIStackView()
+    
+    let aboutButton: UIButton = UIButton()
+    let aboutSeparatorView: UIView = UIView()
+    
+    let termsOfUseButton: UIButton = UIButton()
+    let termsOfUseSeparatorView: UIView = UIView()
+
+    let privacyPolicyButton: UIButton = UIButton()
+    let privacyPolicySeparatorView: UIView = UIView()
+
+    let contactUsButton: UIButton = UIButton()
```

Comment:
if you remember we decided to not use `let variable :UIButton = UIButton()`
please remove `:UIButton` from all views (same for the stack view and other components)

Reply:
Done!